pom.xml

```diff
@@ -103,26 +103,27 @@
     </developers>
 
     <properties>
-        <kotlin.version>1.5.21</kotlin.version>
+        <kotlin.version>1.7.10</kotlin.version>
```

Comment:
Please, check if kotlin 1.7 works fine with java 8

Reply:
Yes it works fine.  Per docs: By default, the Kotlin/JVM compiler produces Java 8 compatible bytecode.